elements/pf-code-block/demo/pf-code-block.html

```diff
@@ -29,13 +29,13 @@ <h2>Expandable</h2>
       connectionConfig:
       url: https://raw.githubusercontent.com/Azure-Samples/helm-charts/master/docs
     </script>
-    <pfe-clipboard slot="actions" copy-from="#expandable-code"></pfe-clipboard>
-  </pfe-code-block>
+    <pf-clipboard slot="actions" copy-from="#expandable-code"></pf-clipboard>
```

Comment:
`<pf-clipboard ...></pf-clipboard>` should be `<pf-clipboard-copy ...></pf-clipboard-copy>`